platform/app/services/connections/create_service.rb

```
@@ -0,0 +1,37 @@
+        destination: connector.workspace.default_analytics_db,
+        name: "#{connector.name} Connection",
+        configuration: {
+          source: connector.configuration,
+          destination: connector.workspace.default_analytics_db.configuration
+        },
+        schedule_type: :manual,
+        status: :created,
+        sync_frequency: :hourly,
+        namespace: :system_defined
+      )
+    end
```

rubocop

[convention] Method has too many lines. [14/10]

(Metrics/MethodLength)